engines/sci/engine/script.cpp

```diff
@@ -955,14 +933,19 @@ void Script::syncLocalsBlock(SegManager *segMan) {
 }
 
 void Script::initializeClasses(SegManager *segMan) {
-	const byte *seeker = 0;
+	SciSpan<const byte> seeker;
 	uint16 mult = 0;
 
 	if (getSciVersion() <= SCI_VERSION_1_LATE) {
-		seeker = findBlockSCI0(SCI_OBJ_CLASS);
+		seeker = _script;
```

Comment:
Yes, `findBlockSCI0` now returns a Span that is the size of the found block, whereas this seeker needs to be able to iterate over the entire script. As far as I could tell, the only thing the `findBlockSCI0` call was doing here was fast forwarding to the first class block.